src/views/base_view.js

```diff
@@ -34,6 +37,11 @@ export default AmpersandView.extend({
     dom.hide(this.el);
   },
 
+  // Adjust the height of the iframe to match the height of the body.
+  adjustHeight () {
+    webChannel.sendAdjustHeight(document.body.offsetHeight);
```

Comment:
Note also that this method doesn't need to be on the base view, since it isn't reused; it should be on the index view.